guides/advanced_guides/settings.md

```diff
@@ -0,0 +1,251 @@
+| ------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| **[synonyms](/guides/advanced_guides/settings.md#synonyms)**                          | List of associated words treated similarly                                       | `{}`                                                                                              |
+| **[stopWords](/guides/advanced_guides/settings.md#stop-words)**                       | List of words ignored by MeiliSearch when present in search queries              | `[]`                                                                                              |
+| **[rankingRules](/guides/advanced_guides/settings.md#ranking-rules)**                 | List of ranking rules sorted by order of importance                              | [A list of ordered built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) |
+| **[distinctAttribute](/guides/advanced_guides/settings.md#distinct-attribute)**       | Search returns documents with distinct (different) values of the given field     | `null`                                                                                            |
+| **[searchableAttributes](/guides/advanced_guides/settings.md#searchable-attributes)** | Fields in which to search for matching query words sorted by order of importance | All attributes found in the documents                                                             |
+| **[displayedAttributes](/guides/advanced_guides/settings.md#displayed-attributes)**   | Fields displayed in the returned documents                                       | All attributes found in the documents                                                             |
+| **[acceptNewFields](/guides/advanced_guides/settings.md#accept-new-fields)**          | Defines whether new fields should be searchable and displayed or not             | `true`                                                                                            |
+
+## Synonyms
+
+A set of words defined for an index. Synonyms are **different words that have the same meaning**, and thus are treated similarly. If either of the associated words is searched, the same results shall be displayed.
+
+`synonyms=<Object>`
+
+- `<Object>` (Object, defaults to `{}`) : `{ <String>: [<String>, <String>, ...], ... }`
+
+  An object that contains words with a list of their associated synonyms.
+
+[Learn more about synonyms](/guides/advanced_guides/synonyms.md)
+
+#### Example
+
+Add settings:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "synonyms": {
+          "wolverine": ["xmen", "logan"],
+          "logan": ["wolverine"]
+      }
+  }'
+```
+
```

Comment:
Can we find a little use case here with for example `black baskets` and `black sneakers` on a e-commerce dataset?